addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -175,6 +199,55 @@ public String getDeviceId() {
     }
 
     @Override
+    public Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException {
+
+        Command OHcommand;
+
```

Comment:
Mmm... It's capitalized because it's an abbreviation for openHAB. Would oH be better? Could be anything really. But the, logical, "command" is out because that's the name of the field in the message we switch on. Which, ironically, is what convertToState() has historically mapped to a state instead of a command :-)